src/pages/home.jsx

```diff
@@ -1,17 +1,40 @@
-import AboutUs from '../components/AboutUs';
-import FAQ from '../components/FaqSection/faq';
-import Hero from '../components/Hero';
-import Navbar from '../components/shared/NavBar';
-import Footer from '../components/shared/marginals/footer';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Button from '../components/shared/Button';
+import AuthContext from '../context/AuthContext';
 
 export default function Home() {
+	const [user, setUser] = useState({});
+
 	return (
-		<div className=''>
-			<Navbar />
-			<Hero />
-			<AboutUs />
-			<FAQ />
-			<Footer />
-		</div>
+		<AuthContext.Provider value={{ user, setUser }}>
+			<div className='container py-6'>
+				<h1 className='text-3xl font-bold'>1999 Reunion Tour</h1>
+				<div className='mt-4'>
+					<h2 className='text-xl font-bold'>How to contribute</h2>
+					<p>
+						1. Store content and other data in <code className='bg-gray-200 p-1 rounded-sm'>data</code> folder.
+					</p>
+					<p>
+						2. Store reusable components in <code className='bg-gray-200 p-1'>components</code> folder.
+					</p>
```

Comment:
Please use the changed home page.

Reply:
done. https://github.com/dscnitrourkela/project-udon/pull/11/commits/6ecba6922d5bff14c9829f16e3552cf8e9b40083